src/main/java/cat/tophat/creepycreepers/common/init/RegistryItem.java

```diff
@@ -39,6 +39,7 @@ public static void registerItems(RegistryEvent.Register<Item> event) {
 	 * some extra setup after Item and EntityType registration completes.
 	 * 
 	 * @author Cadiboo
+	 * @param event Setup event
```

Comment:
That's not really what this is doing. This is only for post attachment of spawn eggs into the map and dispenser behavior.

Reply:
Well technically, the method wouldn't even be called normally because it's an event subscriber. The javadoc exists simply to satiate the build warnings. But I guess I can change the description to be more accurate.